### PR TITLE
SSCS-5178 Bulk scan – ValidationException – Phone number does not mat…

### PR DESCRIPTION
**Pull request by srochani**
…ch pattern

**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java [hunk 1]
```diff
@@ -366,7 +371,14 @@ private void checkExcludedDates(Appeal appeal) {
 
     private boolean isPhoneNumberValid(Appellant appellant) {
         if (appellant != null && appellant.getContact() != null && appellant.getContact().getPhone() != null) {
-            return appellant.getContact().getPhone().matches("^[0-9\\-+ ]{10,17}$");
+            return PhoneNumbersUtil.isValidLandLineNumber(appellant.getContact().getPhone());
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java]: Have you checked this changed validation works with Robotics? The previous regex was copied from the robotics schema. If the edge cases(<10 or >17) still work then fine

Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/validators/SscsCaseValidator.java]: I've checked the PhoneNumberUtil for landline is not accepting more than 10. so revert it back as per Robotics validation.
For Mobile validation using this PhoneNumbersUtil

File: src/main/java/uk/gov/hmcts/reform/sscs/constants/WarningMessage.java [hunk 2]
```diff
@@ -34,6 +34,9 @@
     HEARING_OPTIONS_EXCLUDE_DATES(SscsConstants.HEARING_OPTIONS_EXCLUDE_DATES, "Hearing options exclude dates"),
     APPELLANT_NINO(NINO, "Appellant nino"),
     APPELLANT_PHONE(PHONE, "Appellant phone"),
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/constants/WarningMessage.java]: Think you can probably remove this warning message now? (for appellant phone)

Comment (srochani) [reply src/main/java/uk/gov/hmcts/reform/sscs/constants/WarningMessage.java]: Yes, removed from warning message

**Pull request merged**